tests/vmi_configuration_test.go

```diff
@@ -419,34 +419,43 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				Expect(domXml).To(ContainSubstring("queues='3'"))
 			})
 
-			It("[test_id:1665]should map cores to virtio net queues", func() {
-				if tests.ShouldAllowEmulation(virtClient) {
-					Skip("Software emulation should not be enabled for this test to run")
-				}
+			DescribeTable("should map cores to virtio net queues",
+				func(useAutoAttachedInterfaces bool) {
+					if tests.ShouldAllowEmulation(virtClient) {
+						Skip("Software emulation should not be enabled for this test to run")
+					}
 
-				_true := true
-				_false := false
-				vmi.Spec.Domain.Resources = v1.ResourceRequirements{
-					Requests: kubev1.ResourceList{
-						kubev1.ResourceMemory: resource.MustParse("64M"),
-						kubev1.ResourceCPU:    resource.MustParse("3"),
-					},
-				}
+					_true := true
+					_false := false
+					vmi.Spec.Domain.Resources = v1.ResourceRequirements{
+						Requests: kubev1.ResourceList{
+							kubev1.ResourceMemory: resource.MustParse("64M"),
+							kubev1.ResourceCPU:    resource.MustParse("3"),
+						},
+					}
 
-				vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue = &_true
-				vmi.Spec.Domain.Devices.BlockMultiQueue = &_false
+					vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue = &_true
+					vmi.Spec.Domain.Devices.BlockMultiQueue = &_false
 
-				By("Starting a VirtualMachineInstance")
-				vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
-				Expect(err).ToNot(HaveOccurred())
-				tests.WaitForSuccessfulVMIStart(vmi)
+					if useAutoAttachedInterfaces {
+						vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{}
+						vmi.Spec.Networks = []v1.Network{}
+					}
```

Comment:
Can you instead pass in the values directly? Such that there will be no need to have a condition, just an assignment.

Reply:
Yes but it will be either the empty ones, or what the machine already have, so I found it bit weird (because the latter)
but will change to that